project/AssetCompilation.scala

```diff
@@ -113,7 +117,7 @@ object AssetCompilation {
   val settings = Seq(
     AssetCompilation.SettingsKeys.webpackPath := (Path("node_modules") / ".bin" / "webpack").getPath,
     AssetCompilation.SettingsKeys.npmPath := "yarn",
-    run in Compile <<= (run in Compile) map(killWebpack) dependsOn webpackGenerateTask,
+    run in Compile := {(run in Compile) map(killWebpack) dependsOn webpackGenerateTask},
```

Comment:
This currently leads to `sbt run` not working. Tbh, I'm not sure what the previous semantic was. Any ideas here, @fm3 ?
https://www.scala-sbt.org/1.x/docs/Howto-After-Input-Task.html

Reply:
Which version do you mean breaks sbt run? the old `<<=` or the new `:=`? I’m not sure about the different operators here, either. But I think the intended semantic of this line was that `run` should execute `webpackGenerateTask` before itself and `killWebpack` after itself

Reply:
@rschwanhold already figured out the correct syntax in #3251. Let's merge #3251 first, then I'll merge it in here.